deepfence_worker/tasks/reports/data.go

```diff
@@ -151,11 +160,14 @@ func getVulnerabilityData(ctx context.Context, params sdkUtils.ReportParams) (*I
 		params.Filters.SeverityOrCheckType, params.Filters.AdvancedReportFilters.Masked)
 
 	nodeWiseData := NodeWiseData[model.Vulnerability]{
```

Comment:
This was there before that PR but this could be simplified a lot across `model.Secret`, `model.Vulnerability` and so on